src/components/ActionBar/index.tsx

```diff
@@ -8,12 +8,12 @@ const SPACING = 20
 
 const StyledDiv = styled.div<{ rank: number }>`
   align-items: center;
-  background: ${({ theme }) => theme.colorsDeprecated.white};
-  border: 1px solid ${({ theme }) => theme.colorsDeprecated.gray300};
+  background: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
   border-radius: 4px;
   bottom: ${({ rank }) => SPACING + rank * (HEIGHT + SPACING)}px;
   box-shadow: 0 2px 14px 8px
-    ${({ theme }) => transparentize(0.5, theme.colorsDeprecated.gray200)};
+    ${({ theme }) => transparentize(0.5, theme.colors.neutral.backgroundHover)}; // TODO create shadow named palette ?
```

Comment:
Yes it is planned to have shadow palette, from now we only did it for colors :)

Reply:
```suggestion
    ${({ theme }) => transparentize(0.5, theme.colors.primary.background)};
```

old `gray200` is super tricky as it is not gray but really soft primary color 🤕